settings.gradle

```diff
@@ -105,6 +105,7 @@ if (!System.getenv().containsKey("SUB_BUILD") || System.getenv().get("SUB_BUILD"
     include ':airbyte-integrations:bases:base-java'
     include ':airbyte-integrations:bases:base-java-s3'
     include ':airbyte-integrations:bases:base-normalization'
+    include ':airbyte-integrations:bases:bases-destination-jdbc' // needs to be lexicographically after base-java and base-normalization to avoid race condition
```

Comment:
This is fine and I think eases my concerns about this being a landmine for other teams (and future us) we should maybe at least add a backlog/icebox task to sort through this eventually.

Reply:
Filed ticket here to track https://github.com/airbytehq/airbyte/issues/25251